pkg/mcpfile/types.go

```diff
@@ -72,10 +73,8 @@ type StreamableHTTPConfig struct {
 }
 
 type AuthConfig struct {
-	AuthorizationServers   []string `json:"authorizationServers,omitempty"`   // list of authorization server URLs
-	ScopesSupported        []string `json:"scopesSupported,omitempty"`        // supported OAuth scopes
-	BearerMethodsSupported []string `json:"bearerMethodsSupported,omitempty"` // supported bearer token methods
```

Comment:
this is removed b/c we only support `header` and have the `scopes` inherited as said above?

Reply:
yes